<!DOCTYPE html>
<html lang="en">
   <title>Petros Paplomatas</title>
  <meta charset="UTF-8">
  <meta name="description" content="A brief description of the page">
  <meta name="keywords" content="HTML, CSS, JavaScript">
  <link rel="stylesheet" href="./MyWebsiteCss.css"/>
    
    
     <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@400;700&display=swap" rel="stylesheet">
     
  </head>
  <body>
    <div class="Tabs">
    <input type="button" id="HOME" value="HOME">
    <input type="button" id="Tab1"  value="Publications">
    <input type="button" id="Tab2"  value="Scientific Activities">
    <input type="button" id="Tab3"  value="Software">
    </div>
    
   

    <div id="HOMEContent">


    <div class="Photo_Text">
    <img class="Photo" src="./Paplomatas_Petros.jpg" alt="Photo" />
    <h2 class="Sing">Petros Paplomatas, Diplom(Biomedical Sciences), MSc(Molecular and Biomedical Sciences), MSc
    (Bioinformatic-Neuroinformatic), PhD Candidate</h1>
    <br>
    <br>
     </div>
     <br>
     <p class=Text1> Petros Paplomatas was born in Katerini, Greece in 1987. He received his diploma in 2011 from the Biomedical Sciences Department of the School of Health Sciences of the International Hellenic University-IHU. He received his M.Sc. diploma in Biomedical and Molecular Sciences in the Diagnosis and Treatment of Diseases in 2021 from the Department of Medicine at Democritus University of Thrace. He received his second Master of Science in Bioinformatics and Neuroinformatics in 2023 from the Hellenic Open University. He is now a Ph.D. candidate at Ionian University's department of informatics. His research interests are in the fields of bioinformatics and systems biology. His research focuses on developing novel artificial intelligence or feature selection algorithms for the identification of potential biomarkers for early diagnosis and the development of web interactive applications for this purpose.
     </div>
     
       
<div id="tab1Content">  
<br>  

<h3>
   
       LIST OF PUBLICATIONS  </h3> 
  <br>

    
  
  <ul>
     
 <li> An Ensemble Feature Selection Approach for Analysis and Modeling of Transcriptome Data in Alzheimer’s Disease  <a href="https://doi.org/10.3390/app13042353">https://doi.org/10.3390/app13042353</a>  .</li>
   <br>
  <li> Enhancing the early Alzheimer’s Prediction from single-cell RNA-Sequencing data through Dimensionality Reduction and Feature Selection techniques. <a href="https://doi.org/10.1007/978-3-319-75479-6_65-1">https://doi.org/10.1007/978-3-319-75479-6_65-1</a>  .</li>
  <br>
  <li> Applying SCALEX scRNA-Seq Data Integration for Precise Alzheimer’s Disease Biomarker Discovery  <a href="https://doi.org/10.1007/978-3-031-34171-7_23">https://doi.org/10.1007/978-3-031-34171-7_23</a>  .</li>
  <br>
  <li> Atypical Neurofibromas reveal distinct epigenetic features with proximity to benign peripheral nerve sheath tumor entities <a href="https://doi.org/10.1093/neuonc/noad053">https://doi.org/10.1093/neuonc/noad053</a> .</li>
 <br>
  <li> Are Complete Blood Count Parameters Associated with Climate and Environmental Factors? A Retrospective Study in the General Population of Fokis, Greece (Athens, 2023).  <a href="https://doi.org/10.3390/environsciproc2023026163">https://doi.org/10.3390/environsciproc2023026163"</a> </li>
  <br>
  <li> An interactive web app for the identification of robust genes-level biomarkers for complex diseases.</li>

  
</ul>

</div>
    
<div id="tab2Content">  
<br>  
 
<h3>
 SCIENTIFIC ACTIVITIES
       <br>
 </h3> 
 
<h4 class= h4Tab2> §   Invited Talks at International Conferences  </h4>

 <ul>
       
 
  
  <li> GeNeDis 2022 5th worldwide conference on “Genetics, Geriatrics and Neurodegenerative Diseases Research”. Topic: An ensemble feature selection approach for identifying Alzheimer’s Disease Biomarker	.</li>
  
  <br>
  
  <li> Fosscomm 2022 – Fosscomm 2022 (Free and Open-Source Software Communities Meeting). Topic: A Tool guide for the Identification of more robust genes-level drug targets and biomarkers for complex issues.</li>
  
   <br>
   
<h4 class= h4Tab2> §   Talk Lectures  </h4>
 <br>
    <strong>2023</strong>
 <li>Master of Science course, named "Stem Cells and Regenerative Medicine.	Topic: Introduction to R programming language and development interactive application with Shiny. </li>
  <br>
   <strong>2022</strong>
 <li>Master of Science course, named "Digital Health and Analytics: Clinical Trial and Processes Modeling".	Topic: Development interactive application for Biological analysis with Shiny package. </li>
  <br>
  <strong>2022,2023</strong>
 <li>Master of Science course, named "Biomedical and Molecular Sciences: Diagnosis and Treatment".	Topic: Gene Expression Analysis </li>
 

 

</div>


<div id="tab3Content">  
<br>  

<h3>
       SOFTWARE
         <br>
 </h3> 
<ul>
   <strong>2023</strong>
  <li> <a href="http://212.227.75.194:8447/"> ScGenesElite: </a>A tool-guide for the identification of more robust genes-level drug targets and biomarkers for complex diseases. </li> 
  <p>This application offers a flexible platform for the identification of dominant genes in a single-cell RNA-sequencing (scRNA-seq) dataset that operate as disease biomarkers.
It uses statistical, machine-learning, and cutting-edge feature selection approaches for scRNA-seq data.
The feature selection operation modes include 15 different methodologies, covering a broad range of such approaches. The extracted gene list is further examined for enrichment in various biological and pharmacological features.
A heatmap according to the cell type or the disease state, an enrichment analysis, KEGG molecular pathways, PPI networks, and similarity graphs could be generated from isolated genes to provide a better understanding of the potential biomarkers and molecular interactions that are involved.</p>
  <br>
   <strong>2022</strong>
  <li> <a href="https://cran.r-project.org/web/packages/LDLcalc/index.html">LDLcalc: </a>Calculate and Predict the Low Density Lipoprotein Values. </li> 
  <figure>
    <img class="Photo1" src="./LDLcalcImage.jpeg" alt="LDLcalc C" />  
    <figcaption>Almost 7,000 individuals have downloaded the LDLcalc Package.</figcaption>
    </figure>
  <br>

  
  <br>
</ul>
                           
     
    <script src="./Jscript1.js"></script>



  </body>
</html>
